emukit/examples/simple_gp_model/simple_gp_model.py

```diff
@@ -0,0 +1,121 @@
+from typing import Tuple
+
+import numpy as np
+import scipy.linalg
+import scipy.optimize
+
+from emukit.core.interfaces import IModel
+
+
+class SimpleGaussianProcessModel(IModel):
+    """
+    This model is a Gaussian process with an RBF kernel, with no ARD. It is used to demonstrate uses of emukit,
+    it does not aim to be flexible, robust or fast.
+    """
+    def __init__(self, x: np.ndarray, y: np.ndarray):
+        """
+        :param x: (n_points, n_dims) array containing training features
+        :param y: (n_points, 1) array containing training targets
+        """
+        self.x = x
+        self.y = y
+        self.lengthscale = 1
+        self.kernel_variance = 1
+        self.likelihood_variance = 1
+        self.jitter = 1e-6
+
+    def optimize(self) -> None:
+        """
+        Optimize the three hyperparameters of the model, namely the kernel variance, kernel lengthscale and likelihood
+        variance
+        """
+        def optimize_fcn(x):
```

Comment:
Would you mind renaming x to something else here? I got really confused between this and model's x

Reply:
Yep, changed to `log_hyper_parameters`